server/app/services/export/JsonExporter.java

```diff
@@ -206,4 +228,13 @@ private DocumentContext makeEmptyJsonArray() {
   private DocumentContext makeEmptyJsonObject() {
     return JsonPathProvider.getJsonPath().parse("{}");
   }
+
+  public String getFormattedPhoneNumber(String phoneNumberValue, String countryCode) {
+    try {
+      Phonenumber.PhoneNumber phoneNumber = PHONE_NUMBER_UTIL.parse(phoneNumberValue, countryCode);
+      return PHONE_NUMBER_UTIL.format(phoneNumber, PhoneNumberUtil.PhoneNumberFormat.INTERNATIONAL);
```

Comment:
**question (blocking):** what standard does this formatting follow? will need to provide it in our [API docs](https://docs.civiform.us/it-manual/api/program-applications) (see table of "Question type export values").

Add javadoc for this method that mentions the formatting standard.

Reply:
Thanks for catching this. I have changed it to E164 format.